src/main/webapp/database-tables.js

```diff
@@ -80,12 +80,37 @@ function createTable(name) {
 }
 
 function mainLoad(){
-  showDatabase();
   login();
+  showReason();
 }
 
 function login() {
   fetch("/login").then(response => response.json()).then((user) => {
     document.getElementById("user").innerText = user;
   });
+}
+
+//Note: this method will show the reason assuming the reason is the last thing in the querystring
```

Comment:
We should probably find a way to make sure it works no matter what. Here's a way to do this:

const params = new URLSearchParams(window.location.search);
String reason = params.get('reason');

We should probably also do a check to make sure that the params contains 'reason' - here's a link to more info about this: https://flaviocopes.com/urlsearchparams/

This way should also eliminate a lot of code and make it cleaner!

Reply:
Done, thank you for the suggestion!